packages/eventbridge-client/src/NotificationInput.ts

```diff
@@ -0,0 +1,53 @@
+import { BaseModel } from "@alanszp/validations";
+import { IsEnum, IsNotEmpty, IsString } from "class-validator";
+import { NotificationDetailType } from "./NotificationDetailTypes";
+
+type FixedDetail<T> = {
+  id: string;
+  organizationReference: string;
+  _previousValues: T;
+  lch: string;
+};
+
+type Detail<T> = FixedDetail<T> & T;
```

Comment:
el type que tiene hoy en dia es mas "generico" es:
```suggestion
type Detail = Record<string, unknown>;
```

Si queres definirle un tipo, al genericl podriamos ponerle:

```ts
export interface NotificationInputParams<T extends Record<string, unknown> = Record<string, unknown>> {
```